packages/react-portal/src/components/Portal/usePortalMountNode.ts

```diff
@@ -25,6 +25,17 @@ export const usePortalMountNode = (options: UsePortalMountNodeOptions) => {
     const newElement = targetDocument.createElement('div');
     newElement.setAttribute('class', themeClassName);
     newElement.setAttribute('dir', dir);
+    newElement.style.position = 'fixed';
+    newElement.style.width = '100%';
+    newElement.style.height = '100%';
+    newElement.style.top = '0';
+    newElement.style.right = '0';
+    newElement.style.bottom = '0';
+    newElement.style.left = '0';
+    newElement.style.zIndex = '2';
+    newElement.style.backgroundColor = 'transparent';
+    newElement.style.opacity = '0.9';
```

Comment:
I like the idea of always having an overlay! I think it'd also be a great potential element to make into a semantic close button that can always be accessed by touch screen readers to fix some of the portal a11y issues with them :)